packages/theme-chalk/src/pagination.scss

```diff
@@ -208,6 +208,25 @@
       width: 100px;
     }
   }
+  @include m(large) {
+    .btn-prev,
+    .btn-next,
+    .#{$namespace}-pager li {
+      height: getCssVar('pagination-button-height-large');
+      line-height: getCssVar('pagination-button-height-large');
+      font-size: getCssVar('pagination-font-size-large');
+      min-width: getCssVar('pagination-button-width-large');
+    }
+
+    span:not([class*='suffix']),
+    button {
+      font-size: getCssVar('pagination-font-size-large');
+    }
+
+    .#{$namespace}-select .#{$namespace}-input {
```

Comment:
I think the width here does not need to be changed, keep it consistent with the above.
What do you think? cc @kooriookami @tolking

Reply:
I agree.

Reply:
@webvs2 When size is large, use the default font-size.

Reply:
ok

Reply:
@kooriookami  The 'pagination-font-size-large' field is removed